deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -55,6 +55,16 @@ receivers:
       disk: {}
       filesystem: {}
       network: {}
+  prometheus:
```

Comment:
With this Prometheus receiver enabled here, we don't really need separate metrics collector instance. May not be in this PR but we can just put everything in one place.